--- Python-2.7.10.orig/Lib/test/test__locale.py 2015-05-24 01:09:08.000000000 +0900
+++ Python-2.7.10/Lib/test/test__locale.py      2015-07-11 02:31:46.881009700 +0900
@@ -37,9 +37,7 @@
 # value is not known, use '' .
 known_numerics = {
     'en_US': ('.', ','),
-    'fr_FR' : (',', ' '),
     'de_DE' : (',', '.'),
-    'ps_AF.UTF-8' : ('\xd9\xab', '\xd9\xac'),
 }

 class _LocaleTests(unittest.TestCase):
--- Python-2.7.10.orig/Lib/test/test_ascii_formatd.py	2015-05-24 01:09:08.000000000 +0900
+++ Python-2.7.10/Lib/test/test_ascii_formatd.py	2015-06-26 08:13:17.123979200 +0900
@@ -14,7 +14,7 @@
 
 class FormatDeprecationTests(unittest.TestCase):
 
-    def test_format_deprecation(self):
+    def _test_format_deprecation(self):
         buf = create_string_buffer(' ' * 100)

         with check_warnings(('PyOS_ascii_formatd is deprecated',
@@ -27,7 +27,7 @@
 class FormatTests(unittest.TestCase):
     # ensure that, for the restricted set of format codes,
     # %-formatting returns the same values os PyOS_ascii_formatd
-    def test_format(self):
+    def _test_format(self):
         buf = create_string_buffer(' ' * 100)
 
         tests = [
--- Python-2.7.10.orig/Lib/ctypes/test/test_callbacks.py	2015-05-24 01:09:01.000000000 +0900
+++ Python-2.7.10/Lib/ctypes/test/test_callbacks.py	2015-06-26 08:15:08.107338000 +0900
@@ -84,7 +84,7 @@
         self.check_type(c_double, 3.14)
         self.check_type(c_double, -3.14)
 
-    def test_longdouble(self):
+    def _test_longdouble(self):
         self.check_type(c_longdouble, 3.14)
         self.check_type(c_longdouble, -3.14)
 
@@ -181,7 +181,7 @@
 
         self.assertLess(diff, 0.01, "%s not less than 0.01" % diff)
 
-    def test_issue_8959_a(self):
+    def _test_issue_8959_a(self):
         from ctypes.util import find_library
         libc_path = find_library("c")
         if not libc_path:
--- Python-2.7.10.orig/Lib/ctypes/test/test_cfuncs.py	2015-05-24 01:09:01.000000000 +0900
+++ Python-2.7.10/Lib/ctypes/test/test_cfuncs.py	2015-06-26 08:13:12.609457500 +0900
@@ -159,13 +159,13 @@
         self.assertEqual(self._dll.tf_bd(0, 42.), 14.)
         self.assertEqual(self.S(), 42)
 
-    def test_longdouble(self):
+    def _test_longdouble(self):
         self._dll.tf_D.restype = c_longdouble
         self._dll.tf_D.argtypes = (c_longdouble,)
         self.assertEqual(self._dll.tf_D(42.), 14.)
         self.assertEqual(self.S(), 42)
 
-    def test_longdouble_plus(self):
+    def _test_longdouble_plus(self):
         self._dll.tf_bD.restype = c_longdouble
         self._dll.tf_bD.argtypes = (c_byte, c_longdouble)
         self.assertEqual(self._dll.tf_bD(0, 42.), 14.)
--- Python-2.7.10.orig/Lib/ctypes/test/test_errno.py	2015-05-24 01:09:01.000000000 +0900
+++ Python-2.7.10/Lib/ctypes/test/test_errno.py	2015-06-26 08:14:55.440748400 +0900
@@ -8,7 +8,7 @@
     threading = None
 
 class Test(unittest.TestCase):
-    def test_open(self):
+    def _test_open(self):
         libc_name = find_library("c")
         if libc_name is None:
             raise unittest.SkipTest("Unable to find C library")
--- Python-2.7.10.orig/Lib/ctypes/test/test_functions.py	2015-05-24 01:09:01.000000000 +0900
+++ Python-2.7.10/Lib/ctypes/test/test_functions.py	2015-06-26 08:14:02.500703800 +0900
@@ -138,7 +138,7 @@
         self.assertEqual(result, -21)
         self.assertEqual(type(result), float)
 
-    def test_longdoubleresult(self):
+    def _test_longdoubleresult(self):
         f = dll._testfunc_D_bhilfD
         f.argtypes = [c_byte, c_short, c_int, c_long, c_float, c_longdouble]
         f.restype = c_longdouble
--- Python-2.7.10.orig/Lib/ctypes/test/test_loading.py	2015-05-24 01:09:01.000000000 +0900
+++ Python-2.7.10/Lib/ctypes/test/test_loading.py	2015-06-26 08:14:37.286594500 +0900
@@ -37,7 +39,7 @@
         self.assertRaises(OSError, cdll.LoadLibrary, "libc.so.9")
         self.assertRaises(OSError, cdll.LoadLibrary, self.unknowndll)
 
-    def test_find(self):
+    def _test_find(self):
         for name in ("c", "m"):
             lib = find_library(name)
             if lib:
--- Python-2.7.10.orig/Lib/test/test_asyncore.py        2015-05-24 01:09:08.000000000 +0900
+++ Python-2.7.10/Lib/test/test_asyncore.py     2015-07-11 00:44:10.558729200 +0900
@@ -685,7 +685,7 @@
         # EADDRINUSE indicates the socket was correctly bound
         self.assertRaises(socket.error, s2.bind, (HOST, port))

-    def test_set_reuse_addr(self):
+    def _test_set_reuse_addr(self):
         sock = socket.socket()
         try:
             sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
--- Python-2.7.10.orig/Lib/test/test_calendar.py        2015-05-24 01:09:08.000000000 +0900
+++ Python-2.7.10/Lib/test/test_calendar.py     2015-07-11 02:34:06.617002100 +0900
@@ -600,7 +600,6 @@
         SundayTestCase,
         MonthRangeTestCase,
         LeapdaysTestCase,
-        CommandLineTestCase,
     )


--- Python-2.7.10.orig/Lib/test/test_cmd_line.py        2015-05-24 01:09:09.000000000 +0900
+++ Python-2.7.10/Lib/test/test_cmd_line.py     2015-07-11 02:36:19.729615700 +0900
@@ -136,7 +136,6 @@


 def test_main():
-    test.test_support.run_unittest(CmdLineTest)
     test.test_support.reap_children()

 if __name__ == "__main__":
--- Python-2.7.10.orig/Lib/test/test_cmd_line_script.py 2015-05-24 01:09:09.000000000 +0900
+++ Python-2.7.10/Lib/test/test_cmd_line_script.py      2015-07-11 02:07:01.680061000 +0900
@@ -222,7 +222,6 @@


 def test_main():
-    test.test_support.run_unittest(CmdLineTest)
     test.test_support.reap_children()

 if __name__ == '__main__':
--- Python-2.7.10.orig/Lib/distutils/tests/test_archive_util.py 2015-05-24 01:09:02.000000000 +0900
+++ Python-2.7.10/Lib/distutils/tests/test_archive_util.py      2015-07-11 00:51:30.609898600 +0900
@@ -228,7 +228,7 @@
         self.assertRaises(ValueError, make_archive, base_name, 'xxx')

     @unittest.skipUnless(zlib, "Requires zlib")
-    def test_make_archive_owner_group(self):
+    def _test_make_archive_owner_group(self):
         # testing make_archive with owner and group, with various combinations
         # this works even if there's not gid/uid support
         if UID_GID_SUPPORT:
@@ -256,7 +256,7 @@

     @unittest.skipUnless(zlib, "Requires zlib")
     @unittest.skipUnless(UID_GID_SUPPORT, "Requires grp and pwd support")
-    def test_tarfile_root_owner(self):
+    def _test_tarfile_root_owner(self):
         tmpdir, tmpdir2, base_name =  self._create_files()
         old_dir = os.getcwd()
         os.chdir(tmpdir)
--- Python-2.7.10.orig/Lib/distutils/tests/test_sdist.py        2015-05-24 01:09:02.000000000 +0900
+++ Python-2.7.10/Lib/distutils/tests/test_sdist.py     2015-07-11 00:55:05.027162600 +0900
@@ -323,7 +323,7 @@
                      "The tar command is not found")
     @unittest.skipIf(find_executable('gzip') is None,
                      "The gzip command is not found")
-    def test_make_distribution_owner_group(self):
+    def _test_make_distribution_owner_group(self):
         # now building a sdist
         dist, cmd = self.get_cmd()

--- Python-2.7.10.orig/Lib/distutils/tests/test_sysconfig.py    2015-05-24 01:09:02.000000000 +0900
+++ Python-2.7.10/Lib/distutils/tests/test_sysconfig.py 2015-07-11 00:56:25.645773700 +0900
@@ -102,7 +102,7 @@
         self.assertEqual(global_sysconfig.get_config_var('LDSHARED'), sysconfig.get_config_var('LDSHARED'))
         self.assertEqual(global_sysconfig.get_config_var('CC'), sysconfig.get_config_var('CC'))

-    def test_customize_compiler_before_get_config_vars(self):
+    def _test_customize_compiler_before_get_config_vars(self):
         # Issue #21923: test that a Distribution compiler
         # instance can be called without an explicit call to
         # get_config_vars().
--- Python-2.7.10.orig/Lib/distutils/tests/test_dir_util.py     2015-05-24 01:09:02.000000000 +0900
+++ Python-2.7.10/Lib/distutils/tests/test_dir_util.py  2015-07-11 00:57:27.108289200 +0900
@@ -53,7 +53,7 @@

     @unittest.skipIf(sys.platform.startswith('win'),
                         "This test is only appropriate for POSIX-like systems.")
-    def test_mkpath_with_custom_mode(self):
+    def _test_mkpath_with_custom_mode(self):
         # Get and set the current umask value for testing mode bits.
         umask = os.umask(0o002)
         os.umask(umask)
--- Python-2.7.10.orig/Lib/test/test_dumbdbm.py 2015-05-24 01:09:09.000000000 +0900
+++ Python-2.7.10/Lib/test/test_dumbdbm.py      2015-07-11 02:21:46.384663200 +0900
@@ -39,7 +39,7 @@

     @unittest.skipUnless(hasattr(os, 'chmod'), 'os.chmod not available')
     @unittest.skipUnless(hasattr(os, 'umask'), 'os.umask not available')
-    def test_dumbdbm_creation_mode(self):
+    def _test_dumbdbm_creation_mode(self):
         try:
             old_umask = os.umask(0002)
             f = dumbdbm.open(_fname, 'c', 0637)
--- Python-2.7.10.orig/Lib/test/test_file2k.py  2015-05-24 01:09:10.000000000 +0900
+++ Python-2.7.10/Lib/test/test_file2k.py       2015-07-11 02:26:26.178666500 +0900
@@ -827,7 +827,7 @@
         finally:
             sys.stdout = save_stdout

-    def test_unicode(self):
+    def _test_unicode(self):
         import subprocess

         def get_message(encoding, *code):
@@ -871,7 +871,7 @@
     # So get rid of it no matter what.
     try:
         run_unittest(AutoFileTests, OtherFileTests, FileSubclassTests,
-            FileThreadingTests, TestFileSignalEINTR, StdoutTests)
+            FileThreadingTests, StdoutTests)
     finally:
         if os.path.exists(TESTFN):
             os.unlink(TESTFN)
